operator/cmd/k8s_identity.go

```diff
@@ -117,7 +117,7 @@ func identityGCIteration(ctx context.Context, clientset k8sClient.Clientset) {
 				if identity.Annotations == nil {
 					identity.Annotations = make(map[string]string)
 				}
-				log.WithField(logfields.Identity, identity).Info("Marking identity for later deletion")
+				log.WithField(logfields.Identity, identity.UID).Info("Marking identity for later deletion")
```

Comment:
`logfields.Identity` usually refers to the identity number, which is probably `identity.Name` or similar?

The UID could still potentially be interesting in the case of multiple of the same-named resource, so maybe it can be extended to use `log.WithFields()` and include both the `logfields.Identity` as the identity number, and also `logfields.K8sUID` for the UID.